frontend/src/index.pug

```diff
@@ -151,13 +151,17 @@ html
             span.error-message-box__failed-repo--name {{ errorBlock.repoName }}
             div.error-message-box__failed-repo--reason {{ errorBlock.errorMessage }}
         #summary-charts
-          .summary-charts(v-for="repo of filteredRepos")
-            .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoName }}&nbsp;&nbsp;
+          .summary-charts(v-for="(repo, i) in filteredRepos")
+            .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'")
+              .summary-charts__title--index {{ i+1 }}
+              .summary-charts__title--repo {{ repo[0].repoName }}
               .summary-charts__title--contribution.mini-font(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
             .summary-charts__title(
                 v-if="filterGroupSelection === 'groupByAuthors'",
                 v-bind:class="{ warn: repo[0].name === '-' }"
-            ) {{ repo[0].displayName }} ({{ repo[0].name }})&nbsp;&nbsp;
+            )
+              .summary-charts__title--index {{ i+100 }}
```

Comment:
why is it +100?

Reply:
My bad. Forgot to correct indexing after trying out. I have updated it now.